02_cancer_type_classification/lasso_range_analysis/lasso_range_all.ipynb

```diff
@@ -17,6 +17,7 @@
    "source": [
```

Comment:
<p>is it strange that the test_auroc is so low for CIC-UCEC?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/pancancer-evaluation/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>It's something we've observed before for some genes/cancer types. We think it's because some mutations aren't all that correlated with gene expression in some cancer types (i.e. they're not drivers in that cancer type, or they don't occur frequently enough to train a good model, etc) so they end up overfitting, which is what we see here with good CV performance and bad performance on UCEC samples.</p><p> </p><p>Currently, we're just including/excluding cancer types based on mutation frequency/proportion. We could do something smarter but I think for this study where we have hundreds of genes/cancer types, it's probably fine to have the filter be pretty coarse-grained. A while back we tried filtering for models that outperform a baseline where the gene expression data is permuted, and it didn't affect the overall model selection results that much.</p>